ibm/mas_devops/roles/ocp_setup_mas_deps/tasks/development-catalogs.yml

```diff
@@ -23,17 +23,27 @@
         .dockerconfigjson: "{{ content | join('') | string }}"
   register: result
 
-# 2. Patch the default service account in openshift-marketplace so that it is able to pull from Artifactory
-- name: "Patch default SA on openshift-marketplace"
+# 2. Crete the service accounts in openshift-marketplace so that it is able to pull from Artifactory
```

Comment:
Small typo: `Crete`

Reply:
Also, jsut for my education really, why do we need all the extra service accounts now?

Reply:
1# - Fixed typo. 
2# - Discussed this with @CAIOFCP it seems this is really due a change on OCP 4.8, each catalog source operator now relies on its own SA for the `imagePullSecret` as in:
```
spec:
  nodeSelector:
    kubernetes.io/os: linux
  restartPolicy: Always
  serviceAccountName: ibm-mas-iot-operators
  imagePullSecrets:
    - name: wiotp-docker-local
    - name: ibm-mas-iot-operators-dockercfg-cb7zv
```
So, we just need to create the corresponding SA and patch it with the `wiotp-docker-local` so it authorizes the image pull successfully... haven't removed the `default` SA so it keeps compatibility with OCP 4.6 way.

Reply:
Thanks, I also noticed that we don't have all the app catalogs covered .. so I added a future TODO, but we don't care for this release:

```
  with_items:
    - default
    - ibm-mas-operators
    - ibm-truststore-mgr-operators
    - ibm-mas-manage-operators
    - ibm-mas-iot-operators
    - ibm-sls-operators
    - ibm-data-dictionary-operators
    # TODO: we are missing other application catalogs here!
```